modules/admin_manual/pages/configuration/server/caching_configuration.adoc

```diff
@@ -60,11 +60,11 @@ php -r 'phpinfo();' | grep opcache.enable
 
 === APCu
 
-The easiest cache to use is APCu, because it is a data cache, very fast and nothing needs to be configured. APCu can not be used when needed to run on an external server.
+The easiest cache to use is APCu, because it is a pure data cache, very fast as it is memory based only and _nothing needs to be configured_. APCu can not be used, when needed to run on an external server like in a distributed environment.
```

Comment:
```suggestion
The easiest cache to use is APCu, because it is a pure data cache, very fast as it is memory-based only and _nothing needs to be configured_. APCu can not be used when running on an external server like in a distributed environment.
```